pkg/client/cmd/exec.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/luizalabs/teresa/pkg/client"
+	"github.com/luizalabs/teresa/pkg/client/connection"
+	execpb "github.com/luizalabs/teresa/pkg/protobuf/exec"
+	"github.com/spf13/cobra"
+	context "golang.org/x/net/context"
+)
+
+var execCmd = &cobra.Command{
+	Use:   "exec",
+	Short: "Exec a command on an app replica",
+	Long: `Exec a command on an app replica.
+
+You can execute a command on an app replica (same of current deploy),
+Teresa will collect and stream the stdout of replica until the command ends`,
+	Example: ` $ teresa exec <app-name> -- python manage.py start_job_x -a arg1 -s arg2`,
+	Run:     execCommand,
+}
```

Comment:
Maybe it would be wise to say non-interactive command (shell session won't work for example). Also it would be nice to indent the example based on the other commands.